repository-data/development/src/main/resources/hcm-content/content/documents/corporate-website/publication-system/acceptance-tests/attachment-test.yaml

```diff
@@ -1,5 +1,469 @@
 ---
 /content/documents/corporate-website/publication-system/acceptance-tests/attachment-test:
+  /changed-dataset:
```

Comment:
Was this addition of content to a folder-defining file intentional?

Reply:
I didn't want to have to extract the documents into separate files because I added a number of them. I can extract to a few separate yaml files if you think it necessary? I don't feel strongly either way to be honest